src/control/SConscript

```diff
@@ -8,7 +8,8 @@ from os.path import join, isdir
 from os import urandom
 from binascii import b2a_hex
 
-Import('env', 'prereqs', 'daos_version', 'conf_dir', 'gurt_lib')
+Import('env', 'prereqs', 'daos_version', 'conf_dir',
+       'gurt_lib')
```

Comment:
This is needed because of the way our pylint checking works, the code will work as is but pylint will warn on missing gurt_lib definitions.
If the line length is too long this way then use two Import lines.
```suggestion
Import('env', 'prereqs', 'daos_version', 'conf_dir', 'gurt_lib')
```